### PR TITLE
Question Triggers

### PR DESCRIPTION
**Pull request by dhakim87**
Survey questions can now be triggered by responses to other survey questions.

### CODE REVIEW

File: microsetta_private_api/templates/survey.jinja2 [hunk 1]
```diff
@@ -51,6 +51,20 @@
             // always return false, preventing a traditional post of the form
             return false;
         }
+
+        function set_triggers(model, field)
+        {
+            if ('triggered_by' in field && field.triggered_by != null && field.triggered_by.length > 0)
```
Comment (wasade) [inline microsetta_private_api/templates/survey.jinja2:57]: This may be naive, but does it matter here if this is `!=` or `!==`?

Comment (wasade) [reply microsetta_private_api/templates/survey.jinja2:57]: ..same for the other javascript comparisons

Comment (AmandaBirmingham) [reply microsetta_private_api/templates/survey.jinja2:57]: "JavaScript has both strict and type–converting comparisons. A strict comparison (e.g., ===) is only true if the operands are of the same type and the contents match. The more commonly-used abstract comparison (e.g. ==) converts the operands to the same type before making the comparison."

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators

Comment (AmandaBirmingham) [reply microsetta_private_api/templates/survey.jinja2:57]: The javascript linter always recommends strict comparisons by default ...

Comment (dhakim87) [reply microsetta_private_api/templates/survey.jinja2]: Updated

**Pull request merged**